2025-09-16 11:21:54 - root - INFO - 服务器日志系统初始化完成，日志文件: C:\Users\z0054unn\Documents\Siemens-GitLab\Third-party\third-party-clearance\logs\server_20250916_11h21m54s.log
2025-09-16 11:21:54 - root - INFO - 服务器日志系统初始化完成，日志文件: C:\Users\z0054unn\Documents\Siemens-GitLab\Third-party\third-party-clearance\logs\server_20250916_11h21m54s.log
2025-09-16 11:21:54 - uvicorn.error - INFO - Started server process [24164]
2025-09-16 11:21:54 - uvicorn.error - INFO - Waiting for application startup.
2025-09-16 11:21:54 - back_end.server - INFO - FastAPI应用启动
2025-09-16 11:21:54 - uvicorn.error - INFO - Application startup complete.
2025-09-16 11:22:54 - back_end.server - INFO - the html file is stored at: uploads\LicenseInfo-@automation-core_ac-engineering-ng-2.10.0-2025-07-30_05_57_51.html
2025-09-16 11:22:54 - nodes - INFO - Now we are parsing the original HTML File
2025-09-16 11:22:54 - nodes - INFO - Successfully parsed!
2025-09-16 11:22:54 - nodes - INFO - Now we are parsing the original PCR file.
2025-09-16 11:22:54 - nodes - INFO - Now we are reviewing the components list
2025-09-16 11:22:54 - nodes - INFO - We have 3 to review
2025-09-16 11:23:09 - nodes - INFO - We have reviewed one component
2025-09-16 11:23:25 - nodes - INFO - We have reviewed one component
2025-09-16 11:23:38 - nodes - INFO - We have reviewed one component
2025-09-16 11:23:38 - nodes - INFO - Completely Reviewed.
2025-09-16 11:23:38 - nodes - INFO - Now we are collecing special licenses such as GPL
2025-09-16 11:23:38 - nodes - INFO - Now checking the reviewed risks
2025-09-16 11:23:38 - nodes - INFO - We have 3 licenses to check
2025-09-16 11:23:38 - nodes - INFO - Successfully combined 3 license and risk records
2025-09-16 11:23:48 - nodes - INFO - We have checked one component
2025-09-16 11:24:00 - nodes - INFO - We have checked one component
2025-09-16 11:24:10 - nodes - INFO - We have checked one component
2025-09-16 11:24:10 - nodes - INFO - finished checking, now we are checking the dependecies
2025-09-16 11:24:10 - nodes - INFO - now we start analyzing depencies between components
2025-09-16 11:24:19 - nodes - INFO - Now we have checked dependency of one component
2025-09-16 11:24:19 - utils.itemFilter - INFO - 开始筛选组件，共 1 个组件待处理
2025-09-16 11:24:19 - utils.itemFilter - INFO - 处理组件 1/1
2025-09-16 11:24:19 - utils.itemFilter - INFO - 正在检查组件: @ngrx/store 17.2.0
                            ⇧
2025-09-16 11:24:19 - utils.itemFilter - INFO - 组件许可证列表: ["Apache-2.0", "CC-BY-4.0", "MIT"]
2025-09-16 11:24:19 - utils.itemFilter - INFO - 风险分析中的许可证: ["Apache-2.0", "CC-BY-4.0", "MIT", "3: MIT⇧"]
2025-09-16 11:24:19 - utils.itemFilter - INFO - 筛选完成，保留了 1 个需要凭证的组件
2025-09-16 11:24:19 - nodes - INFO - finished checking, now we are starting the chat...
2025-09-16 11:24:22 - nodes - INFO - Initialized session successfully, waiting for the bot to start conversation
2025-09-16 11:24:22 - nodes - INFO - Starting Chatting...
2025-09-16 11:24:22 - SpecialConsiderationHandler - INFO - No items will be found.
2025-09-16 11:24:22 - SpecialConsiderationHandler - ERROR - content_handler_factory must be provided for ChapterGenerationHandler
2025-09-16 11:24:22 - SpecialConsiderationHandler - ERROR - subcontent_factory must be provided for ChapterGeneration
2025-09-16 11:24:22 - back_end.services.chat_service - INFO - we have found risk bot when initializing the chat service!
2025-09-16 11:24:22 - back_end.services.chat_manager - WARNING - now we start with checking: component
2025-09-16 11:24:25 - back_end.server - INFO - now we initialized status as: OEMing
2025-09-16 11:30:07 - uvicorn.error - ERROR - Exception in ASGI application
Traceback (most recent call last):
  File "C:\Users\z0054unn\AppData\Local\Programs\Python\Python313\Lib\site-packages\uvicorn\protocols\http\h11_impl.py", line 403, in run_asgi
    result = await app(  # type: ignore[func-returns-value]
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        self.scope, self.receive, self.send
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "C:\Users\z0054unn\AppData\Local\Programs\Python\Python313\Lib\site-packages\uvicorn\middleware\proxy_headers.py", line 60, in __call__
    return await self.app(scope, receive, send)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\z0054unn\AppData\Local\Programs\Python\Python313\Lib\site-packages\fastapi\applications.py", line 1054, in __call__
    await super().__call__(scope, receive, send)
  File "C:\Users\z0054unn\AppData\Local\Programs\Python\Python313\Lib\site-packages\starlette\applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "C:\Users\z0054unn\AppData\Local\Programs\Python\Python313\Lib\site-packages\starlette\middleware\errors.py", line 187, in __call__
    raise exc
  File "C:\Users\z0054unn\AppData\Local\Programs\Python\Python313\Lib\site-packages\starlette\middleware\errors.py", line 165, in __call__
    await self.app(scope, receive, _send)
  File "C:\Users\z0054unn\AppData\Local\Programs\Python\Python313\Lib\site-packages\starlette\middleware\cors.py", line 93, in __call__
    await self.simple_response(scope, receive, send, request_headers=headers)
  File "C:\Users\z0054unn\AppData\Local\Programs\Python\Python313\Lib\site-packages\starlette\middleware\cors.py", line 144, in simple_response
    await self.app(scope, receive, send)
  File "C:\Users\z0054unn\AppData\Local\Programs\Python\Python313\Lib\site-packages\starlette\middleware\exceptions.py", line 62, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "C:\Users\z0054unn\AppData\Local\Programs\Python\Python313\Lib\site-packages\starlette\_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "C:\Users\z0054unn\AppData\Local\Programs\Python\Python313\Lib\site-packages\starlette\_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "C:\Users\z0054unn\AppData\Local\Programs\Python\Python313\Lib\site-packages\starlette\routing.py", line 714, in __call__
    await self.middleware_stack(scope, receive, send)
  File "C:\Users\z0054unn\AppData\Local\Programs\Python\Python313\Lib\site-packages\starlette\routing.py", line 734, in app
    await route.handle(scope, receive, send)
  File "C:\Users\z0054unn\AppData\Local\Programs\Python\Python313\Lib\site-packages\starlette\routing.py", line 288, in handle
    await self.app(scope, receive, send)
  File "C:\Users\z0054unn\AppData\Local\Programs\Python\Python313\Lib\site-packages\starlette\routing.py", line 76, in app
    await wrap_app_handling_exceptions(app, request)(scope, receive, send)
  File "C:\Users\z0054unn\AppData\Local\Programs\Python\Python313\Lib\site-packages\starlette\_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "C:\Users\z0054unn\AppData\Local\Programs\Python\Python313\Lib\site-packages\starlette\_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "C:\Users\z0054unn\AppData\Local\Programs\Python\Python313\Lib\site-packages\starlette\routing.py", line 73, in app
    response = await f(request)
               ^^^^^^^^^^^^^^^^
  File "C:\Users\z0054unn\AppData\Local\Programs\Python\Python313\Lib\site-packages\fastapi\routing.py", line 301, in app
    raw_response = await run_endpoint_function(
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    ...<3 lines>...
    )
    ^
  File "C:\Users\z0054unn\AppData\Local\Programs\Python\Python313\Lib\site-packages\fastapi\routing.py", line 212, in run_endpoint_function
    return await dependant.call(**values)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\z0054unn\Documents\Siemens-GitLab\Third-party\third-party-clearance\back_end\server.py", line 186, in chat
    chat_flow = session['chat_flow']
                ~~~~~~~^^^^^^^^^^^^^
KeyError: 'chat_flow'
2025-09-16 11:31:18 - uvicorn.error - INFO - Shutting down
2025-09-16 11:31:18 - uvicorn.error - INFO - Waiting for application shutdown.
2025-09-16 11:31:18 - back_end.server - INFO - FastAPI应用关闭
2025-09-16 11:31:18 - uvicorn.error - INFO - Application shutdown complete.
2025-09-16 11:31:18 - uvicorn.error - INFO - Finished server process [24164]
